notebooks/233-blip-visual-language-processing/233-blip-visual-language-processing.ipynb

```diff
@@ -143,7 +143,7 @@
     }
```

Comment:
<p>Should we clear cells output?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1129/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Outputs should be preserved for documentation building. Perhaps here we can use <code>-q</code> parameter for <code>pip install</code> to hide the output.  </p>

Reply:
<p>Output can remain here</p>